### PR TITLE
Update .travis.yml for CI support

### PR DESCRIPTION
**Pull request by kiryteo**
Added 'coala --ci' command to travis.yml file, to import coala into Travis CI build.
Fixes https://github.com/coala/coala-vs-code/issues/22

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -51,6 +51,7 @@ script:
   - sed -i.bak '/bears = GitCommitBear/d' .coafile
   # Server side tests.
   - coverage run $(which behave) ./tests/server.features
+  - run coala --ci
```
Comment (gaocegege) [inline .travis.yml]: run is not a command, it should be `coala --ci`, IMO

**Comment by gaocegege** [comment]
And please read http://api.coala.io/en/latest/Developers/Newcomers_Guide.html to follow the convention of our community :-)

**Comment by kiryteo** [comment]
I've changed the command, do check if possible

**Comment by gaocegege** [comment]
@kiryteo 

Thanks for your contribution, your code works for me but the commit message could be better: http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html

Please refine the commit message and it will help you to be a good contributor :-)

**Comment by kiryteo** [comment]
I've added a little description for the commit, thanks for the guidance though.!

**Comment by gaocegege** [comment]
Great! Could you push your commit to review?

**Comment by kiryteo** [comment]
I'll just close this PR and create a new one with all things done rightly.. Is that okay.?

**Comment by gaocegege** [comment]
@kiryteo I don't think so, you could run `git reset HEAD~1` in your master branch, and the command will reset your latest commit. Then you could commit the changes again and `git push --force` to override the commit 2f12bb3

**Comment by gaocegege** [comment]
There is no need to file a new PR since it does the same thing as this one.

**Comment by kiryteo** [comment]
okay, I'll do that soon.

**Comment by kiryteo** [comment]
I haven't used git from terminal for this changes.. I've just added the necessary command to my forked repository and made the pull request.

**Comment by gaocegege** [comment]
Well, I think it is better to learn git basics in terminal since it is the basis of our project :-)

**Comment by kiryteo** [comment]
Sure, I'll work on it.

**Comment by gaocegege** [comment]
Go ahead and if you have any question you could ask here or our gitter channel https://gitter.im/coala/coala

**Comment by kiryteo** [comment]
Actually, I asked about this issue on gitter firstly but couldn't get the
required response, so I mentioned it directly here. Anyway, I would follow
the instructions and do the same.

**Comment by gaocegege** [comment]
Hi @kiryteo 

Is there any update?

**Comment by kiryteo** [comment]
@gaocegege  I have added the commit for review, please have a look. 
Sorry for the delay.

**Comment by gaocegege** [comment]
Hi, please squash the commit and repush, then we could merge it.

**Comment by kiryteo** [comment]
@gaocegege Please do have a look.. I think now it should be okay..

**Comment by gaocegege** [comment]
Hi, @kiryteo 

There are two commits now and I think we could squash it to one :)

**Comment by kiryteo** [comment]
@jayvdb I have squashed the commits.

**Comment by gaocegege** [comment]
Please resolve the conflicts :smile:

**Comment by kiryteo** [comment]
@gaocegege  so should I remove the '- coverage run -a -m unittest discover -s tests' part. my commit doesn't conflicts with that..

**Comment by kiryteo** [comment]
@gaocegege  resolved conflict, please have a look.

**Comment by gaocegege** [comment]
> I suggest you come onto gitter and ask someone to help you step by step to fix this.

PTAL

**Comment by jayvdb** [comment]
Closed due to inactivity.

**Pull request closed without merging**